rllib/algorithms/cql/cql.py

```diff
@@ -207,14 +197,6 @@ def training_step(self) -> ResultDict:
         else:
             train_results = multi_gpu_train_one_step(self, train_batch)
 
-        # Update replay buffer priorities.
-        update_priorities_in_replay_buffer(
```

Comment:
I'm not sure if we have the time realistically and the CQL implementation is not currently being used by any customers. I'd recommend making this a p1 or p2

Reply:
This will enable CRR usage and confirmation of whether we are on the right track, I agree that we should benchmark these once there is some sign of life on an actual use-case.